src/ascend.ts

```diff
@@ -229,36 +234,59 @@ function isInValhalla(): boolean {
 }
 
 /**
- * Hops the gash, perming no skills
+ * Hops the gash, perming no skills by default
  *
- * @param path path of choice, as a Path object--these exist as properties of Paths
- * @param playerClass Your class of choice for this ascension
- * @param lifestyle 1 for casual, 2 for softcore, 3 for hardcore. Alternately, use the Lifestyle enum
- * @param moon Your moon sign as a string, or the zone you're looking for as a string
- * @param consumable From the astral deli. Pick the container item, not the product.
- * @param pet From the astral pet store.
- * @param permOptions Options for perming during a player's stay in Valhalla
- * @param permOptions.permSkills A Map<Skill, Lifestyle> of skills you'd like to perm, ordered by priority.
- * @param permOptions.neverAbort Whether the ascension shouold abort on failure
+ * @param inputValues Configuration for the ascension
+ * @param inputValues.path path of choice, as a Path object--these exist as properties of Paths
+ * @param inputValues.playerClass Your class of choice for this ascension
+ * @param inputValues.lifestyle 1 for casual, 2 for softcore, 3 for hardcore. Alternately, use the Lifestyle enum
+ * @param inputValues.kolGender An entry from the KolGender enum: 1 for male, 2 for female (sorry that it's limited to those). Defaults to 2 or the corresponding value for defaultGenderOverride pref (which should be 'male' or 'female')
+ * @param inputValues.moon Your moon sign as a string, or the zone you're looking for as a string
+ * @param inputValues.consumable From the astral deli. Pick the container item, not the product. Defaults to astral six-pack, provide $item`none` for nothing.
+ * @param inputValues.pet From the astral pet store.
+ * @param inputValues.permOptions Options for perming during a player's stay in Valhalla
+ * @param inputValues.permOptions.permSkills A Map<Skill, Lifestyle> of skills you'd like to perm, ordered by priority.
+ * @param inputValues.permOptions.neverAbort Whether the ascension should abort on failure
  */
-export function ascend(
-  path: Path,
-  playerClass: Class,
-  lifestyle: Lifestyle,
-  moon: InputMoonSign,
-  consumable: Item | undefined = $item`astral six-pack`,
-  pet: Item | undefined = undefined,
-  permOptions?: { permSkills: Map<Skill, Lifestyle>; neverAbort: boolean }
-): void {
+export function ascend(inputValues: {
```

Comment:
I know that you got this name from me, but we should probably call this `options` instead of `inputValues`